packages/server/postgres/queries/getMeetingSeriesByIds.ts

```diff
@@ -0,0 +1,24 @@
+import MeetingSeriesId from 'parabol-client/shared/gqlIds/MeetingSeriesId'
+import getPg from '../getPg'
+import {
+  getMeetingSeriesByIdsQuery,
+  IGetMeetingSeriesByIdsQueryResult
+} from './generated/getMeetingSeriesByIdsQuery'
+
+export interface MeetingSeries extends IGetMeetingSeriesByIdsQueryResult {}
+
+const getMeetingSeriesByIds = async (meetingSeriesIds: readonly string[]) => {
+  const meetingSeriesResults = await getMeetingSeriesByIdsQuery.run(
+    {ids: meetingSeriesIds.map((id) => MeetingSeriesId.split(id))},
+    getPg()
+  )
+
+  return meetingSeriesResults.map((meetingSeries) => {
+    return {
+      ...meetingSeries,
+      id: MeetingSeriesId.join(meetingSeries.id)
```

Comment:
-1 I'll use this comment as a catch-all for the meetingSeriesId:
- There are 4 calls to `MeetingSeriesId.join`. 1 in the mutation, 2 in the Meeting resolver, 1 in the DB query. It feels strange because `TeamPromptMeeting.meetingSeriesId` has it in the resolver, but `MeetingSeries.id` doesn't have it in the resolver (it uses the default resolver). Makes it hard to know where in the code it's the prefixed GUID vs. the DBID (string vs. number). For example, if we added `Team.meetingSeries`, the developer would have to know to map over the `meetingSeries` to give them all GUIDs.

I propose the following:
- Add a `MeetingSeries.id` resolver that inputs a DBID and outputs a GUID
- remove the other calls to `MeetingSeriesId.join` except for `TeamPromptMeeting.meetingSeriesId`

That way everything gets transformed on the way out & we get a nice clean DBID to work with everywhere else. Open to other ideas!

Reply:
Yea, this sounds like the best way forward.  Updated